README.md

```diff
@@ -1,17 +1,148 @@
-# Vizro
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/title.svg" width="200"/>
+</p>
 
-[![Python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue.svg)]()
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)]()
+<div align="center" markdown="1">
 
+[![Python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue.svg)](https://pypi.org/project/vizro/)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/mckinsey/vizro/blob/main/LICENSE.md)
+
+</div>
+
+<div align="center" markdown="1">
+
+<a href="https://vizro.readthedocs.io/en/latest/" target="_blank">Documentation </a> |
+<a href="https://vizro.readthedocs.io/en/latest/pages/tutorials/first_dashboard/" target="_blank">Get Started </a> |
+<a href="http://vizro.mckinsey.com/" target="_blank">Live Demo </a>
+
+</div>
+
+---
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/example_screens.png" width="700"/>
+</p>
+
+<p align="center">
+<font size="+2">
+<b>
+Visual Intelligence. Beautifully engineered
+</b>
+</font>
+</p>
+
+<p align="center">
+<font size="+1">
 Vizro is a toolkit for creating modular data visualization applications
+</font>
+</p>
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/tech_logos.png" width="270"/>
+</p>
+
+<p align="center">
+<font size="+1">
+View a live <a href="http://vizro.mckinsey.com/" target="_blank">interactive demo </a>
+</font>
+</p>
+
+<p align="center">
+<a href="https://vizro.readthedocs.io/en/latest/">
+<img border="0" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/docs_button.svg" width="94" height="32">
+</a>
+&nbsp;&nbsp;&nbsp;&nbsp
+<a href="https://vizro.readthedocs.io/en/latest/pages/tutorials/first_dashboard/">
+<img border="0" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/get_started_button.svg" width="94" height="32">
+</a>
+</p>
+
+## What is Vizro?
+
+<p align="center">
+<font size="+1">
+Rapidly self-serve the assembly of customised dashboards in minutes - without the need for advanced coding or design experience - to create flexible and scalable, Python enabled data visualization applications
+</font>
+</p>
```

Comment:
Can we keep this left-aligned? As everything in this section is left-aligned as well. I think the center alignment makes sense for the part above, but here it looks a bit out of place as the rest of the section here is left-aligned